src/eXpansion/Framework/Core/Helpers/ChatNotification.php

```diff
@@ -59,4 +58,21 @@ public function sendMessage($messageId, $to = null, $parameters = [])
 
         $this->connection->chatSendServerMessage($message, $to);
     }
+
+    /**
+     * Return messageId with arguments as a string
+     * Usage: used for retrieving partials for chat messages
+     *  * defaults to English locale, without parameters
+     *
+     * @param $messageId
+     * @param array $parameters
+     * @param string $locale
+     * @return mixed
+     */
+    public function getMessage($messageId, $parameters = [], $locale = "en")
```

Comment:
Can't you simply use the transation service instead of the chatMessage service to get the message?